src/Apps/Sell/Routes/IntroRoute.tsx

```diff
@@ -1,70 +1,91 @@
 import * as React from "react"
-import { Box, Button, Flex, SkeletonBox, Spacer, Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Flex,
+  Join,
+  Separator,
+  Image,
+  Spacer,
+  Text,
+} from "@artsy/palette"
 import { useRouter } from "System/Hooks/useRouter"
 import { AppContainer } from "Apps/Components/AppContainer"
 import { SubmissionHeader } from "Apps/Sell/Components/SubmissionHeader"
 
+export const introData = [
+  {
+    number: "1",
+    title: "Tell us about your work",
+    description:
+      "Start by adding an artist from our list of high demand artists. Include information such as year, medium, dimensions and materials.",
+    image: "https://files.artsy.net/images/01_submission_artwork.png",
+  },
+  {
+    number: "2",
+    title: "Upload artwork images",
+    description:
+      "Improve your chances of selling by including photographs of the front, back, frame, signature, and other details.",
+    image: "https://files.artsy.net/images/02_submission_upload.png",
+  },
+  {
+    number: "3",
+    title: "Complete submission",
+    description:
+      "Your work will be submitted to an Artsy advisor who will assess whether your work is eligible and help guide you on next steps.",
+    image: "https://files.artsy.net/images/03_submission_advisor.png",
```

Comment:
this image and the one above it look tiny, we should upload bigger ones to make sure they don't look blurry

Reply:
Don't we have a service to upload images like this, getting them processed with Gemini, etc.? I just cannot remember the name. Maybe this would be the right approach here 🤔

Reply:
We can crop or resize using utils within Force ([example](https://github.com/artsy/force/blob/6fe3e9857205feab903aef6a297277a710ed00f5/src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/FAQSWA.tsx#L7-L10)).

I went ahead and uploaded high quality images here via tools.artsy.net:

https://files.artsy.net/images/01_submission_intro_artwork.png
https://files.artsy.net/images/02_submission_intro_upload.png
https://files.artsy.net/images/03_submission_intro_advisor.png

Reply:
fyi — this is where you can usually find the original image within Figma if you click down the layers enough: ![Screenshot 2024-06-18 at 12 26 12](https://github.com/artsy/force/assets/123595/ada0dbbd-d6b7-4ab5-bfe9-d13bf4b3ca96)

Reply:
Thanks!

Reply:
I think it would be best to export cropped images, it's difficult with out Image component to adjust (can get hacky)